### DISCUSSION TITLE
国际版 HyperOS BootLoader 解锁规则出来了,有搞头吗？

### DISCUSSION THREAD

**Discussion by YaKun9** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#50] [General]
国际版现在不需要答题，目前好像也没有等级限制。
途径：xiaomi community  app 

![b8308df679b11b45be639ea34a4f188](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/131952790/0e12a1f4-b521-4f27-9703-4234ac0f7998)
![8294c82408fc10eb029ca6082adf11c](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/131952790/ecab7b0f-6e10-4d4d-a499-6de40f468ffb)

**Comment by MlgmXyysd** [comment]
昨天晚上有，现在没了

**Reply by YaKun9** [reply]
据说申请了,国内版的权限就没了，还是没法搞

**Reply by 88558639** [reply]
国内版什么权限没了？

**Reply by YaKun9** [reply]
> 国内版什么权限没了？

各种权限，比如你加入的各种内测。
但这点只是看到网友们在说，不确定实际上什么样子。
小米应该很快就会进行封堵的。

**Reply by 88558639** [reply]
bl之后还在乎内测么，直接刷三方包了呀

**Reply by YaKun9** [reply]
据酷安消息称：已经封锁了，不用尝试了😂

**Comment by ghost** [comment]
突发奇想啊，全程开全局模式，挂美国节点，注册一个美国小米账号，然后登陆海外版小米社区，申请解锁，看看能不能通过

**Reply by MlgmXyysd** [reply]
有号没用，绑定不了国行手机
